SU-SWS/stanford_media
# Url objects on image templates

# READY FOR REVIEW

# Summary
- Fixed linkit support when using responsive image styles
- Remove custom alt text field in favor of one provided by contrib space.
- Created update hook for the migration of the alt text fields.

# Needed By (Date)
- soon

# Urgency
- medium

# Steps to Test
*Use suhumsci site in the following steps*
1. stay on `8.x-1.x` branch
1. sync mathematics `blt drupal:sync --site=mathematics --partial`
1. review the button in the footer area "Make a Gift" see that it has the following issues:
  * Link href is `/node/16` instead of the url alias of the page
  * no `title` attribute on the link
1. Edit the block
1. double click the button.
1. verify the issue: there's an `Alternate text` and an `Alternative text` field
1. checkout this branch
1. `drush @mathematics.local updb -y`
1. verify the issues above are corrected and the image/link looks and functions correctly.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
